bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ember-cli-materialize",
   "dependencies": {
-    "ember": "~1.10.0",
+    "ember": "~1.12.0",
```

Comment:
I like to encourage people to develop on the oldest ember version we aim to support. This way, nobody is surprised when their 1.12.0 computed property macro syntax causes CI to fail.